prow/plugins/trigger/pull-request.go

```diff
@@ -30,13 +30,22 @@ import (
 )
 
 func handlePR(c Client, trigger plugins.Trigger, pr github.PullRequestEvent) error {
-	if len(c.Config.Presubmits[pr.PullRequest.Base.Repo.FullName]) == 0 {
-		return nil
-	}
-
 	org, repo, a := orgRepoAuthor(pr.PullRequest)
 	author := string(a)
 	num := pr.PullRequest.Number
+
+	baseSHA, err := c.GitHubClient.GetRef(org, repo, "heads/"+pr.PullRequest.Base.Ref)
+	if err != nil {
+		return fmt.Errorf("failed to get baseSHA: %v", err)
+	}
+	presubmits, err := c.Config.GetPresubmits(c.GitClient, org+"/"+repo, baseSHA, pr.PullRequest.Head.SHA)
+	if err != nil {
+		return fmt.Errorf("failed to get presubmits: %v", err)
+	}
+	if len(presubmits) == 0 {
```

Comment:
Same concern about this increasing token usage although this is only 1 token per event.

What do you think about having a list of orgs and repos to enable `prow.yaml` files in the config? That would be an easy way to save API tokens, avoid cloning repos without `prow.yaml` files, and seems like it might be generally useful info.